test/CMakeLists.txt

```diff
@@ -118,10 +118,15 @@ if(NRN_ENABLE_PYTHON AND PYTEST_FOUND)
     endif()
   endif()
   if (NRN_ENABLE_MPI)
+    get_filename_component(MPIEXEC_NAME ${MPIEXEC} NAME)
```

Comment:
@alexsavulescu : could you add one line comment about why we do that? I am sure if someone else will look at this, they will wonder why someone is doing this :)

Reply:
Ah, forgot to copy the comment from my fork.